README.md

```diff
@@ -68,21 +71,21 @@ In the above case, where the namespace is `openshift-must-gather-b9fpw` and the
 pod name is `must-gather-mxc7k` we would enter the container with:
 
 ```bash
-$ oc -n openshift-must-gather-b9fpw rsh must-gather-mxc7k
+oc -n openshift-must-gather-b9fpw rsh must-gather-mxc7k
 ```
 
 And then if we were debugging a bash script called `gather_trigger_gmr`, we
 would run it in debug mode:
 
 ```bash
-sh-5.1# bash -x /usr/bin/gather_trigger_gmr
```

Comment:
-1: This prefix was written on purpose to reflect that this is running within the container.